org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/utils/FilesUtils.java

```diff
@@ -153,13 +153,16 @@ public static void saveToFile(String content, Path outFile) throws IOException {
 			writer.write(content);
 		}
 		// Make sure it's not executable
-		try {
-			Files.setPosixFilePermissions(outFile, PosixFilePermissions.fromString("rw-r-----"));
-		} catch (UnsupportedOperationException e) {
-			// The associated file system does not support the PosixFileAttributeView,
-			// ignore the error
-			LOGGER.log(Level.SEVERE,
-					"The associated file system '" + outFile + "' + does not support the PosixFileAttributeView", e);
+		// Windows doesn't support marking files as executable/not-executable
+		if (!Platform.isWindows) {

```

Comment:
I wonder if correct fix should be just:

`outFile.toFile().setExecutable(false);`

which should work for any OS. @fbricon what do you think about that?

Reply:
yeah I guess it should work. Unit tests will confirm ;-)